Infrastructure-Support-of-Publication-Process.md

```diff
@@ -0,0 +1,35 @@
+# Infrastructure Support for DevRel Publication Process
+
+This process provides instructions for setting up and updating the [EOS Dev Portal](docs.eosnetwork.com). 
+
+## Build a New Document Repository
+
+The following link provides instructions on how to build a new document repository. Use this procedure to initiate a new document repository for the EOS dev portal (infrastructure), when necessary.  
+
+[How to Build a New Repository](https://github.com/eosnetworkfoundation/devdocs/blob/main/docs/How-To-Do-A-Full-Release.md) 
+
+## How to Automate Documentation 
+
+The following link provides instructions on how to automate documentation for the EOS Dev Portal.  
+
+[How to Autopmate Documentation](https://github.com/eosnetworkfoundation/devdocs/blob/main/README.md) 
+
+## Make Changes to the EOS Dev Portal Navigation 
+
+The following link provides instructions on how to make changes to the EOS Dev Portal navigation, such as the left sidebar site index, the right sidebar document table of contents, and the footer.  
+
+[How to Make Changes to the EOS Dev Portal Navigation](https://github.com/eosnetworkfoundation/devdocs/blob/main/docs/SiteNavigation.md) 
+
+## How to Generate Documents 
+The following link provides instructions on how to generate or regenerate documents on the EOS Dev Portal.
+
+[How to Generate Documents](https://github.com/eosnetworkfoundation/devdocs/blob/main/docs/GeneratingDocuments.md) 
+
+## How to Search Content (TBD) 
+
+[How to Search Content](https://github.com/eosnetworkfoundation/devdocs/blob/main/docs/site-search.md) 
+
+(Add available indexed search criteria/setups). 
+## How to Publish Content
```

Comment:
This section is unfinished

If we don't have enough information about it we can leave them out for now

Reply:
Done